www/src/pages/blog/astro-018.astro

```diff
@@ -0,0 +1,105 @@
+---
+import { Markdown, Prism } from 'astro/components';
+import BlogHead from '../../components/BlogHead.astro';
+import BlogHeader from '../../components/BlogHeader.astro';
+import BlogPost from '../../components/BlogPost.astro';
+import BlockQuote from '../../components/BlockQuote.astro';
+import GoogleAnalytics from '../../components/GoogleAnalytics.astro';
+import Note from '../../components/Note.astro';
+import Shell from '../../components/Shell.astro';
+import { mediaQueries } from '../../config.js';
+
+let title = 'Astro 0.18 Release';
+let description = `Astro 0.18 is our biggest release since Astro launch. It includes a new responsive hydrator, named slots, global stylesheets, and two new renderers. Ready on to learn more on this release.`;
+let publishDate = 'Tuesday, July 27 2021';
+let author = 'matthew';
+let heroImage = '/social.jpg';
+let permalink = 'https://astro.build/blog/astro-0-18';
+let lang = 'en';
+---
+
+<html lang={ lang ?? 'en' }>
+  <head>
+    <BlogHead title={title} description={description} permalink={permalink} />
+    <style global>
+      img {
+        max-width: 100%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author={author} heroImage={heroImage} publishDate={publishDate}>
+        <Markdown>
+          A little over a month ago, the first public beta for Astro was released to the world. Since then, we have been fixing bugs and gathering your feedback on what to tackle next. Today, we are excited to announce the release of some of our most requested features.
+
+          We are excited to introduce Astro v0.18, featuring:
+
+          * __[Responsive partial hydration:](#responsive-component-hydration)__ Hydrate components with CSS media queries.
+          * __[Named slots:](#named-slots)__ Support multiple content entrypoints inside of Astro components.
+          * __[Solid.js support:](#solid-support)__ Use [Solid.js](https://www.solidjs.com/) components in Astro.
+          * __[Lit support:](#solid-support)__ Use [Lit SSR](https://lit.dev/) to get server-side rendering for web components.
+          * [`<style global>` support](https://docs.astro.build/guides/styling#overview), [GitHub syntax highlighting](https://twitter.com/n_moore/status/1417881860051509250) and a [shiny new docs site.](https://docs.astro.build/)
+
+          ## Responsive partial hydration
+
+
+          <img src="/assets/blog/astro-018/responsive-hydration.png" alt="Code example that shows off using the new client:media hydrator." />
```

Comment:
I like this new image, but the component is for a DesktopSidebar and the media query is for < 700px! I'll update it. Curious how you got the file size down though.

Reply:
Doh! I always confuse those two.

I used ImageOptim! @drwpow showed it to me a while back, it's great and simple.